nextflow/modules/annotation/TransferAnnotationsToMatrixTable/main.nf

```diff
@@ -14,14 +14,9 @@ process TransferAnnotationsToMatrixTable {
         """
         set -ex
 
-        tar --no-same-owner -xf ${annotations}
```

Comment:
The previous Stage doesn't Tar this directory, and the current stage doesn't need to un-tar it